spec/models/article_spec.rb

```diff
@@ -629,6 +630,21 @@ def assert_sets_trigger(art)
       end
     end
 
+    describe "merge_with" do
+      it "should merge two articles" do
+        article1 = Factory.create(:article, :title => "First Article")
```

Comment:
There is a lot of setup and expectation checking in this test, consider separating with some whitespace for readability purposes